how-to-write-a-transformer.md

```diff
@@ -1,516 +1 @@
-            }
-            ctx.putType(updatedInput)
-        }
-    }
-
-    private addVersionedInputToUpdateInput(
-        ctx: TransformerContext,
-        typeName: string,
-        versionInput: string,
-    ) {
-        return this.addVersionedInputToInput(
-            ctx,
-            ModelResourceIDs.ModelUpdateInputObjectName(typeName),
-            versionInput
-        )
-    }
-
-    private addVersionedInputToDeleteInput(
-        ctx: TransformerContext,
-        typeName: string,
-        versionInput: string,
-    ) {
-        return this.addVersionedInputToInput(
-            ctx,
-            ModelResourceIDs.ModelDeleteInputObjectName(typeName),
-            versionInput
-        )
-    }
-
-    private addVersionedInputToInput(
-        ctx: TransformerContext,
-        inputName: string,
-        versionInput: string,
-    ) {
-        const input = ctx.getType(inputName)
-        if (input && input.kind === Kind.INPUT_OBJECT_TYPE_DEFINITION) {
-            const updatedFields = [
-                ...input.fields,
-                makeInputValueDefinition(versionInput, makeNonNullType(makeNamedType("Int")))
-            ]
-            const updatedInput = {
-                ...input,
-                fields: updatedFields
-            }
-            ctx.putType(updatedInput)
-        }
-    }
-
-    private enforceVersionedFieldOnType(
-        ctx: TransformerContext,
-        typeName: string,
-        versionField: string,
-    ) {
-        const type = ctx.getType(typeName)
-        if (type && type.kind === Kind.OBJECT_TYPE_DEFINITION) {
-            const versionFieldImpl = type.fields.find(f => f.name.value === versionField)
-            let updatedField = versionFieldImpl
-            if (versionFieldImpl) {
-                const baseType = getBaseType(versionFieldImpl.type)
-                if (baseType === 'Int' || baseType === 'BigInt') {
-                    // ok.
-                    if (versionFieldImpl.type.kind !== Kind.NON_NULL_TYPE) {
-                        updatedField = {
-                            ...updatedField,
-                            type: makeNonNullType(versionFieldImpl.type),
-                        }
-                    }
-                } else {
-                    throw new TransformerContractError(`The versionField "${versionField}" is required to be of type "Int" or "BigInt".`)
-                }
-            } else {
-                updatedField = makeField(versionField, [], makeNonNullType(makeNamedType('Int')))
-            }
-            const updatedFields = [
-                ...type.fields,
-                updatedField
-            ]
-            const updatedType = {
-                ...type,
-                fields: updatedFields
-            }
-            ctx.putType(updatedType)
-        }
-    }
-}
-```
\ No newline at end of file
+<meta http-equiv="refresh" content="0; URL='https://github.com/aws-amplify/docs/blob/master/js/graphql.md'" />
```

Comment:
same as above, make sure these redirect to the actual compiled site:

https://aws-amplify.github.io/aws-amplify/docs/js/graphql